order/controller.go

```diff
@@ -421,20 +399,67 @@ func (c *Controller) Account() (model.Account, error) {
 	return c.exchange.Account()
 }
 
-func (c *Controller) Position(pair string) (asset, quote float64, err error) {
-	return c.exchange.Position(pair)
+func (c *Controller) Position(pair string) (*model.Position, bool) {
+	//return c.exchange.Position(pair)
+
+	if !c.mockTrading {
```

Comment:
I didn't understand the reason of this flag

Reply:
I added this flag because in Backtesting/PaperWallet, we need to handle internally some logic that in real trading is handled by the exchange. For example,  if one of the positions are liquidated in backtesting, we need to create a market order in opposite direction. In this way the wallet is updated and we'll see on the chart where the position is liquidated. In real trading, the position is liquidated by the exchange automatically, so we don't have to create a new order for this.